### ISSUE TITLE
CustomTensorboard and ValidationEvaluator callbacks

### ISSUE THREAD

**Issue by jhaggle** [kurapan/EAST#57]
In the CustomTensorboard class you set log_dir=FLAGS.checkpoint_path + '/train'

In the ValidationEvaluator class you set validation_log_dir=FLAGS.checkpoint_path + '/val'

Can you explain how those tensorboard objects are supposed to differ? Because when I run
tensorboard --logdir='path/val' and tensorboard --logdir='path/val' I get graphs with the same names, i.e. loss, pred_geo_map_loss and pred_score_map_loss.  Also, the graphs don’t have the same content, se the images here: [graphs](https://imgur.com/a/80BA9SL). The first graph comes from the path + '/train' -directory and to second graph from the path + '/val' -directory.

(Not so important in this connection, but tensorflow board from the  path + '/train' -directory also contains learning rate and small text weight graphs).  

Furthermore, you are normally able to see not only training loss and valuation loss, but also training ACCURACY and valuation ACCURACY. Is there now way to see this in your implementation??

**Comment by germanebr** [comment]
Hi, I know that this question is not related to your problem, but I've been having problems running the training script, and I see that you were able to run it. I was wondering if you could help me running the script, since every time that I solve an error, another one pops out immediately.
